config.toml

```diff
@@ -3,7 +3,7 @@ address = ":8081"
 api_prefix = "/api/v1/"
 api_spec_file = "openapi.json"
 debug = true
-auth = false
```

Comment:
Why did you change this?

Reply:
For testing, I forgot to restore it before committing. Fixed